xc7/primitives/common_slice/common_slice.pb_type.xml

```diff
@@ -477,7 +539,8 @@
       </metadata>
     </mux>
     <mux name="CARRY_DI0" input="COMMON_SLICE.AO5 COMMON_SLICE.AX" output="CARRY4_VPR.DI0" >
-      <delay_constant max=".105e-9" in_port="AX" out_port="CARRY4_VPR.DI0" />
```

Comment:
This is not exactly right.  The previous delay was **not** the CY mux delay, but instead the difference between CARRY4_[ABCD]X and CARRY4_O5.

Reply:
I have removed those changes from the PR. This will need some more logic to calculate those delays on the fly. I don't want to stall this PR, so I'll open a new one with those